ecommerce/frontend/src/components/UserReview.js

```diff
@@ -0,0 +1,128 @@
+import React, { useEffect, useState } from 'react'
+import { Rate, Pagination, Spin } from 'antd'
+import { Link } from 'react-router-dom'
+import './UserReview.less'
+import { sleep } from '../utils'
+import { api } from '../api'
+
+export const UserReview = ({ productId = 1, pageSize = 10 }) => {
+    const [isLoading, setIsLoading] = useState(true)
+    const [currentPage, setCurrentPage] = useState(1)
```

Comment:
pay attention when passing the currentPage to backend since antd expects pages to be 1-indexed while backend expects 0-indexed, just letting you know.